drivers/sensor/lis2dh/lis2dh.c

```diff
@@ -440,6 +512,7 @@ int lis2dh_init(const struct device *dev)
  */
 
 #define LIS2DH_DEVICE_INIT(inst)					\
+	PM_DEVICE_DT_INST_DEFINE(inst, lis2dh_pm_action);		\
 	DEVICE_DT_INST_DEFINE(inst,					\
 			    lis2dh_init,				\
 			    NULL,					\
```

Comment:
you're passing `NULL` in the PM field, so this device will not have PM. Please check other drivers and make sure this implementation is tested accordingly, this patch can't work as it is now.